x/concentrated-liquidity/pool_test.go

```diff
@@ -91,6 +111,16 @@ func (s *KeeperTestSuite) TestInitializePool() {
 				s.App.PoolManagerKeeper.SetParams(s.Ctx, params)
 			}
 
+			if len(test.unrestrictedPoolCreatorWhitelist) > 0 {
+				params := s.App.ConcentratedLiquidityKeeper.GetParams(s.Ctx)
+				params.UnrestrictedPoolCreatorWhitelist = test.unrestrictedPoolCreatorWhitelist
+				fmt.Println("setting")
+				fmt.Println(params.UnrestrictedPoolCreatorWhitelist)
+				s.App.ConcentratedLiquidityKeeper.SetParams(s.Ctx, params)
+				fmt.Println("getting")
+				fmt.Println(s.App.ConcentratedLiquidityKeeper.GetParams(s.Ctx).UnrestrictedPoolCreatorWhitelist)
```

Comment:
nit: prints but I saw them being removed in a follow-up PR so good with keeping them until then